nss-evening-cohort-8/nutshell-chufas
# Ml display messages

#17 
@shanedwilson @Feven90 @ripalpate 
How to test 
- login to site
- message board appears
- messages with your uid should only have edit and delete buttons (added 1 message for each of yall)
- when you resize window, scrollbar appears but message input stays at the top
- input and submit button dont have functionality

- im hoping there is a way to get the userName from the uid. I have only figured out how to get it for the current user. asking zoe tonight because firebase has weird admin issues
